typings/externs.d.ts

```diff
@@ -15,6 +15,13 @@ declare global {
     code?: string;
   }
 
+  /** Make properties K in T optional. */
+  type MakeOptional<T, K extends keyof T> = {
```

Comment:
Does this actually get used anywhere? Seems like you might have opted for `Exclude` instead